### DISCUSSION TITLE
400 Bad Request

### DISCUSSION THREAD

**Discussion by thecowmilk** [netbox-community/netbox#12124] [Help Wanted!]
I have seen other similar problems and they solved it by a changing to `ALLOWED` hosts. Where can I find such configuration?

**Comment by thecowmilk** [comment]
Changed `ALLOWED_HOSTS = ['']` to my IP and still it won't open. Restrated NGINX Multiple times.

**Reply by markkuleinio** [reply]
https://docs.netbox.dev/en/stable/configuration/required-parameters/#allowed_hosts
